pkgs/top-level/all-packages.nix

```diff
@@ -19954,6 +19954,11 @@ with pkgs;
 
   teeworlds = callPackage ../games/teeworlds { };
 
+  tengine = callPackage ../servers/http/tengine {
+    stdenv = overrideCC stdenv gcc6;
```

Comment:
Is this necessary? Did you try to build with default gcc7 ?

Reply:
Error build with gcc7.